traitsui/wx/tree_editor.py

```diff
@@ -1256,9 +1256,12 @@ def _new_actions(self, node, object):
                 name = class_name
             if not factory:
                 factory = klass
-            def perform_add(object):
+            def perform_add(object, factory, prompt):
                 self._menu_new_node(factory, prompt)
-            items.append(Action(name=name, on_perform=perform_add))
+            on_perform = partial(
```

Comment:
`partial` is undefined here (and in the original PR that this was backported from)